### ISSUE TITLE
[RFE] GUI: Custom sort order of widgets on Dashboard 

### ISSUE THREAD

**Issue by fraenki** [opnsense/core#597]
I'd like to be able to change the sort order of the items on the dashboard. I've seen some GUIs allowing to move widgets around to resort them. This would be a nice (and IMHO useful) addition to the dashboard.

**Comment by fichtner** [comment]
It's already in the code, but the javascript to pull it off is missing. This and 3-column support has been discussed a few times, yet I doubt it'll be ready for 16.1.

**Comment by AdSchellevis** [comment]
The code behind the dashboard probably needs some thorough recoding, if we want to keep this in the current form I think we need to improve code quality first before we start adding new functionality here.

**Comment by fabianfrz** [comment]
duplicate of #210

**Comment by fichtner** [comment]
Closed the older one in favour of this vivid discussion :)

**Comment by o0JoeCool0o** [comment]
This would be great as pfsense has this capability and I have migrated to opnsense and lost this functionality. opnsense interface is amazing compared to pf but I think this is vital.

**Comment by b2000699** [comment]
** b2000699 commented 7 minutes ago
Dashboard: Autorefresh loses scrolling position [#821](https://github.com/opnsense/core/issues/821) 
**

> fichtner commented a minute ago:
> Agreed. I would like to move this to #597 as well as I suspect fixing this now will double the amount of work in this area for something that has been the default behaviour (even though iffy even for me) for a long time -- could you add a note there and close this? :)

16.1.6 with Firefox 44.0.2:
Every few seconds the dashboard scrolls up to the first line. I guess that the cause for this is that the dashboard is getting refreshed by some widget.

This feels broken, since it is highly annoying when having scrolled down and monitored some widget. It would be good, if the dashboard would remember the scrolling position after refreshing itself and thus not change its scrolling position so not to annoy the user.

**Comment by fichtner** [comment]
Things are happening. :)
